foundation/foundation-configuration-store/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  #%L
+  AIOps Foundation::AIOps MDA
+  %%
+  Copyright (C) 2021 Booz Allen
+  %%
+  This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+  #L%
+  -->
+
+<Configuration>
+	<Appenders>
+		<Console name="Console" target="SYSTEM_OUT">
+			<PatternLayout
+				pattern="%highlight{%d{HH:mm:ss.SSS} [%t] %-5level %logger{36}.%M():%L - %msg%n}{FATAL=red blink, ERROR=red, WARN=yellow bold, INFO=grey, DEBUG=green bold, TRACE=blue}" />
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="com.boozallen.aiops" level="info" additivity="false ">
+			<AppenderRef ref="Console" />
+		</Logger>
+		<Root level="info">
```

Comment:
A: If we're setting the root logger to `info` and the package-specific logger to `info`, that seems redundant. Do we want to set this to `error` or remove the package-specific logger?

Reply:
great catch!